src/openrct2/util/SawyerCoding.h

```diff
@@ -31,15 +31,15 @@ enum
 
 enum
 {
-    FILE_VERSION_MASK = (3 << 0),
-    FILE_VERSION_RCT1 = (0 << 0),
-    FILE_VERSION_RCT1_AA = (1 << 0),
-    FILE_VERSION_RCT1_LL = (2 << 0),
-
-    FILE_TYPE_MASK = (3 << 2),
-    FILE_TYPE_TD4 = (0 << 2),
-    FILE_TYPE_SV4 = (1 << 2),
-    FILE_TYPE_SC4 = (2 << 2)
+    FILE_VERSION_MASK = 0b00000011,
+    FILE_VERSION_RCT1 = 0b00000000,
```

Comment:
Again, I appreciate the masks. I think these versions and types should not be defined as either bitmask or flag, though. I think they should be decimal instead.

(This should not be taken as a strong objection; just voicing some thoughts.)